app/view/panel/application/StartViewController.js

```diff
@@ -2,18 +2,29 @@ Ext.define('MoMo.admin.view.panel.application.StartViewController', {
     extend: 'Ext.app.ViewController',
     alias: 'controller.momo-application-start-view',
 
+    /**
+     * 
+     */
     renderMap: function() {
         var me = this;
 
         me.setOlMap();
-        //me.addMapCrossHair();
         me.registerOnMapMoveEnd();
 
     },
 
+    /**
+     * 
+     */
     setOlMap: function() {
         var me = this,
             view = me.getView(),
+            viewModel = me.getViewModel(),
+            defaultData = viewModel.getData(),
+            zoom = defaultData.mapZoom,
+            x = defaultData.mapCenter.x,
+            y = defaultData.mapCenter.y,
+            projCode = defaultData.mapProjection,
             olMap;
 
         olMap = new ol.Map({
```

Comment:
Yes. I had the same thought. I think this will follow in one of the next PRs :wink: